main.py

```diff
@@ -32,11 +35,18 @@
 BALL_NOMARK    =  0
 BALL_DISTANCE  = 30
 
+# Colours - Playfield, Player1, Player2
+COLOURS = [WHITE, BLACK, RED]
+
 # Playingfield coordinations - Level 1
-COORDS = [  (0,0, BLACK),                                           (4,0, RED),
-            (0,1, BLACK), (1,1, WHITE), (2,1, WHITE), (3,1, WHITE), (4,1, RED),
-            (0,2, BLACK),               (2,2, WHITE),               (4,2, RED),
-            (0,3, BLACK),                                           (4,3, RED)]
+COORDS = [  (0,0),                      (4,0),
+            (0,1), (1,1), (2,1), (3,1), (4,1),
+            (0,2),        (2,2),        (4,2),
+            (0,3),                      (4,3)]
+
+# Start and goal constellation - Level 1
+START = [1, 2, 1, 0, 0, 0, 2, 1, 0, 2, 1, 2]
+GOAL =  [2, 1, 2, 0, 0, 0, 1, 2, 0, 1, 2, 1]
```

Comment:
```suggestion

START = [1,          2, 
         1, 0, 0, 0, 2,
         1,    0,    2, 
         1,          2]
 
GOAL =  [2,          1,
         2, 0, 0, 0, 1,
         2,    0,    1,
         2,          1]
```
Or variant with macros:
```suggestion
START = [P1,             P2, 
         P1,  N,  N,  N, P2,
         P1,      N,     P2, 
         P1,             P2]
 
GOAL =  [P2,             P1,
         P2,  N,  N,  N, P1,
         P2,      N,     P1,
         P2,             P1]

```

Reply:
applied as suggested